client/src/app/containers/Register/index.jsx

```diff
@@ -21,43 +49,52 @@ const Landing = () => {
   const onInputChange = event =>
     setFormData({ ...formData, [event.target.name]: event.target.value });
 
+  const classes = useStyles();
+
   return (
-    <div className='max-w-sm m-auto mt-12'>
-      <h1 className='text-4xl'>Welcome to Cribbage!</h1>
-      <p>Play cribbage against your friends online. Get started now!</p>
-      <form onSubmit={onSubmitForm}>
-        <input
-          name='id'
-          onChange={onInputChange}
-          value={formData.id}
-          placeholder='Username'
-          required
-          className='mt-2 form-input'
-        ></input>
-        <input
-          name='name'
-          onChange={onInputChange}
-          value={formData.name}
-          placeholder='Display name'
-          required
-          className='mt-2 form-input'
-        ></input>
-        <p className='mt-1 text-xs text-gray-600'>
-          Already have an account?{' '}
-          <span>
-            <Link to='/' className='hover:text-gray-500 hover:underline'>
-              Log in here.
-            </Link>
-          </span>
-        </p>
-        <input
-          type='submit'
-          value='register'
-          className='mt-1 btn btn-primary'
-        ></input>
-      </form>
-    </div>
+    <Container component='main' maxWidth='sm'>
+      <CssBaseline />
+      <div className={classes.paper}>
+        <Typography component='h1' className={classes.title}>
+          Welcome to Cribbage!
+        </Typography>
+        <p>Play cribbage against your friends online. Get started now!</p>
+        <form className={classes.form} onSubmit={onSubmitForm}>
+          <TextField
```

Comment:
I think there are mui components that can make this look really good:

https://github.com/joshprzybyszewski/cribbage/pull/71/files#diff-1e7bed6495e491de3d0b171b40f8ca21R121-R132

Reply:
I followed the example straight from the doc page for `TextField`: https://material-ui.com/components/text-fields/

And it looks pretty good - do you disagree?